ui/src/app/views/homepage/component.html

```diff
@@ -1,80 +1,135 @@
-<div class="front-page">
-    <div class="background-image">
-    <div class="rectangle"></div>
-    <div class="invisible_rectangle">
-        <div class="tip-title">{{tipHeadline}}</div>
-        <div class="tip-title tip-subtitle">{{tipSubtitle}}</div>
-        <div class="tip-title tip-description">{{tipDetail}}</div>
-        <div *ngIf="tipButton" class="learn-more-button" (click)="bugReportComponent.reportBug();">{{tipButton}}</div>
-        <div class="profile-box">
-            <div style="float: left; width: 50%">
-                <div class="profile-title">
-                    COMPLETE YOUR PROFILE
+<div class="background-image">
+    <div class="row">
+        <div *ngFor="let card of cardDetails">
+            <div class="col">
+                <div class="card double {{card.position}}">
+                    <div class="row">
+                        <img class="icon topIcon" src={{card.icon}}>
+                    </div>
+                    <div class="row heading">
+                        {{card.title}}
+                    </div>
+                    <div class="row">
+                        <label class="text">{{card.text}}</label>
+                    </div>
+                    <div class="row heading">
+                        COMING SOON...
+                    </div>
                 </div>
-                <div class="profile-title profile-tips">
-                    You can quickly:
-                    <li>Add your institution</li>
-                    <li>Add your role</li>
-                    <li>Update something else</li>
+            </div>
+        </div>
+    </div>
+    <div *ngIf="firstTimeUser">
+        <div class="row">
+            <div class="col">
+                <div class="card double left">
+                    <div class="row">
+                        <img class="icon analyzeIcon" src="/assets/icons/Analyze.svg">
+                    </div>
+                    <div class="row analyzeHeading">Analyze and Publish with Workspaces
+                    </div>
+                    <div class="row">
+                        <label class="text">Dolor sit amet consectetuer adipiscing sed diam
+                            euismod tincidunt ut laoreet dolore.</label>
+                    </div>
+                    <div class="row heading">
+                        <button class="btn btn-homepage" (click)="addWorkspace()">CREATE WORKSPACE</button>
```

Comment:
If we can, I would have the text here be regularly cased (E.g., 'Create Workspace') and then do a `text-transform: uppercase` on it in styling.